cc-cli/Program.cs

```diff
@@ -283,6 +324,51 @@ static void Main(string[] args)
             }
         }
 
+        private static bool UpdateIsAvailableConversation(string testResponse = "n")
+        {
+            var updated = false;
+
+            _logger.Log("An update is available. Continue with update? ('y/yes' or 'n/no')", MessageType.DisplayInfo);
```

Comment:
Do we need to tell the user how long the update process takes or what to expect if they update? Maybe not if this process is super quick and seamless... Just curious.

Reply:
It is pretty quick, I believe we have some future stories to address informing the user of what the "Release Notes" are.
We would pull those from the notes section on each release and display them as they are on GitHub.

Reply:
Sounds reasonable!